tool/teleport/common/teleport.go

```diff
@@ -41,8 +41,10 @@ import (
 
 // same as main() but has a testing switch
 //   - cmdlineArgs are passed from main()
-//   - testRun is 'true' when running under an integration test
-func Run(cmdlineArgs []string, testRun bool) (executedCommand string, conf *service.Config) {
+//   - dontStart, if set to 'true', initializes config and auxiliary endpoints
+//     but does not start teleport which might be useful for tests and custom
+//     pre-start handlers
+func Run(cmdlineArgs []string, dontStart bool) (executedCommand string, conf *service.Config) {
```

Comment:
I would make start a boolean and will make an option

Run(Options{
   Args: []
   InitOnly: false
})